scripts/tests/api_compare/docker-compose.yml

```diff
@@ -60,6 +61,43 @@ services:
       timeout: 10m
       retries: 3
       start_period: 10m
+  forest-rpc-ready:
```

Comment:
Why not just update `healthcheck` of `forest`?

Reply:
This could work, but I prefer to have a dedicated service for now.

For example, `forest-wallet-import` doesn't need to wait until Ethereum mapping is "up to date".